_components/figure/_figure.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Rename to `_figure` to be absolutely sure this file does not render or get included in the sitemap